src/Frontend/Pages/Products/Products.jsx

```diff
@@ -48,8 +60,13 @@ function Products() {
                 <div>Up to ₹15000 off on Exchange</div>
               </div>
               <div className="product-actions">
-                <Button type="secondary" data="Add To Cart" />
-                <Button type="orange" data="Buy Now" />
+                <Button type="secondary" data="Add To Wishlist" />
+                <Button
+                  navigate={null}
+                  clickFun={() => handleAddToCart(_product)}
+                  type="orange"
+                  data="Add To Cart"
```

Comment:
After adding to cart, this button's text can be changed to something like "Added".

Reply:
I have Implemented Snackbar for that you can check in deploy preview